### ISSUE TITLE
Not available via hacs in HA 99.2 or manually

### ISSUE THREAD

**Issue by mkotek** [daenny/climate_group#6]
For some reason, the component cannot be added via custom repositoey in hacs. What's more, when created manually and uploaded into custom_xomponents, it still does not work in 99.2. The Configurator check does not allow it to be used.

**Comment by jokerigno** [comment]
maybe you are using 0.0.1 version who works for HA version before 0.96 (when climate component has been rebuilt?

BTW @daenny can you harmonize versions?

I've installed master version who works good on 0.99.2 (thanks) but I always see message to update component (while I know is the latest version avaliable).

Thank you

**Comment by mkotek** [comment]
I do not see the component in hacs at all. So I have downloaded the file from Github, uploaded it to custom_components/climate_group as climate.py and restarted HA. Next I have added climate: entry in my configuration and check config says, it is invalid as the component cannot be found. I have already climate working with z-wave definition and I am not duplicating the entry, but adding additional platform.
